src/App.tsx

```diff
@@ -6,14 +6,19 @@ import { Product } from './components/Product';
 import { SearchBar } from './components/SearchBar';
 import { TextInput } from './components/TextInput';
 import { lightTheme } from './themes';
+import { Route } from 'react-router-dom';
+import StoreHomePage from './pages/StoreHomePage';
 
```

Comment:
그냥 제 개인적인 취향인데 
여기 임포트문들
```suggestion
import React, { useState } from 'react';
import { DefaultTheme, ThemeProvider } from 'styled-components';
import { Route } from 'react-router-dom';

import { CustomButton } from './components/Button';
import { Product } from './components/Product';
import { SearchBar } from './components/SearchBar';
import { TextInput } from './components/TextInput';

import StoreHomePage from './pages/StoreHomePage';

import { useAlert } from './hooks/useAlert';

import { lightTheme } from './themes';
```

이런식으로 모듈-컴포넌트-페이지-hooks-theme 이런식으로 띄워서 분리해주시면 좀 더 가독성 좋을거가타요